CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Records breaking changes from major version bumps
 
+## 17.0.0
+
+Removes the customer benefits record email from G-Cloud 11 manifest, as this is no longer used by CCS.
```

Comment:
Pretty sure the email is only used in the Buyer FE's Direct Award flow templates? Let's say so here, so that people don't have to go digging all over again when they pull this into the other FE apps.